### PR TITLE
fix cdc consistent config

### PR DESCRIPTION
**Pull request by CharlesCheung96**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by shichun-0415** [comment]
@CharlesCheung96 Please involve a tech review. Thanks.

**Review by nongfushanquan — APPROVED** [review]

File: ticdc/ticdc-sink-to-mysql.md [hunk 1]
```diff
@@ -105,8 +105,8 @@ level = "eventual"
 # 单个 redo log 文件大小，单位 MiB，默认值 64，建议该值不超过 128。
 max-log-size = 64
 
-# 刷新或上传 redo log 至 S3 的间隔，单位毫秒，默认 1000，建议范围 500-2000。
-flush-interval = 1000
+# 刷新或上传 redo log 至 S3 的间隔，单位毫秒，建议该参数 >= 2000。
+flush-interval = 2000
```
Comment (shichun-0415) [inline ticdc/ticdc-sink-to-mysql.md:109]: Is this config the same as `flush-interval` in [Replicate Data to Storage Services](/ticdc/ticdc-sink-to-cloud-storage.md#optional-parameters)?
image

Comment (CharlesCheung96) [reply ticdc/ticdc-sink-to-mysql.md:109]: No, this is configuration in sink module. And we fix the configuration of redo module in this pr.

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**